fpga/lib/core.cpp

```diff
@@ -64,9 +53,14 @@ std::list<std::shared_ptr<Core>> CoreFactory::make(Card *card,
 
     allIps.push_back({vlnv, ipName});
   }
+  return allIps;
+}
 
+std::list<IpIdentifier>
+CoreFactory::reorderIps(std::list<IpIdentifier> allIps) {
```

Comment:
does this split really make sense? Do we ever need an "unordered" list? We do not want external code to use the list returned from `parseVLNV` above, or do we?

Reply:
Making the ordering of ips optional is beneficial, because later refactors might be able to completely remove ordered initialization dependencies.